src/examples/text-inputs/Bare.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Field, Label, Input } from '@zendeskgarden/react-forms';
+
+const Example = () => (
+  <Field>
+    <Label>Plant</Label>
+    <Input isBare placeholder="Plants" />
+  </Field>
+);
+
+export default Example;
```

Comment:
Note to self: changing `Bare.tsx` to `InputBare.tsx` would increase consistent naming across files. See avatar and button examples for consistent naming. This naming change would apply to all Text input example files.